src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/PdlUtil.kt

```diff
@@ -10,7 +10,7 @@ import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 
 val secureLogger: Logger = LoggerFactory.getLogger("secureLogger")
-private val logger: Logger = LoggerFactory.getLogger(PdlClient::class.java)
+val logger: Logger = LoggerFactory.getLogger(PdlClient::class.java)
```

Comment:
Hvorfor endre denne? 
```suggestion
private val logger: Logger = LoggerFactory.getLogger(PdlClient::class.java)
```

Reply:
Fordi at den brukes i feilsjekkOgReturnerData, og dermed kan den ikke aksesseres hvis den er private ?